src/openrct2/ride/Ride.cpp

```diff
@@ -5528,10 +5516,10 @@ int32_t ride_get_entry_index(int32_t rideType, int32_t rideSubType)
     return subType;
 }
 
-StationObject* ride_get_station_object(const Ride* ride)
+StationObject* Ride::GetStationObject() const
```

Comment:
```suggestion
const StationObject* Ride::GetStationObject() const
```
Makes probably more sense, no one should modify the result.